src/pages/auth/components/sign-up-form.tsx

```diff
@@ -1,23 +1,74 @@
 import { Box, Divider, Typography } from "@mui/material";
 import { CredentialResponse, GoogleLogin } from "@react-oauth/google";
-import React from "react";
+import { FetchBaseQueryError } from "@reduxjs/toolkit/query";
+import React, { useCallback, useEffect } from "react";
 import { useTranslation } from "react-i18next";
+import { useNavigate } from "react-router-dom";
 
 import { BaseButton, CustomFormGroup } from "~/components/index.ts";
+import { AppRoute } from "~/libs/constants/app-route.ts";
+import { useAddUserGoogleMutation } from "~/redux/auth/auth-api.ts";
+import { setTokens, setUser } from "~/redux/auth/auth-slice.ts";
+import { useAppDispatch } from "~/redux/hooks.ts";
 import theme from "~/theme.ts";
 
 import { useSignUpForm } from "../hooks/index.ts";
 import { AuthLinks, SignInLink } from "./index.ts";
 import { StyledFormContainer } from "./styles.ts";
 
-type Properties = {
-	onClick: (credentialResponse: CredentialResponse) => void;
-};
-
-const SignUpForm: React.FC<Properties> = ({ onClick }: Properties) => {
+const SignUpForm: React.FC = () => {
+	const dispatch = useAppDispatch();
+	const navigate = useNavigate();
 	const { t } = useTranslation();
 	const { control, errors, handleFormSubmit, isLoading, serverError } =
 		useSignUpForm();
```

Comment:
why its on new line?

Reply:
Because of the rules eslint[prettier/prettier]
Should we delete it?

Reply:
We have 	"printWidth": 80,
If we change it now to 100 or 120, we should format all the code. I leave this question open.